### PR TITLE
chore(nxdev): improve colours on website according to latest design

### PR DESCRIPTION
**Pull request by meeroslav**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: nx-dev/nx-dev/tailwind.config.js [hunk 1]
```diff
@@ -11,13 +11,11 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        black: '#222222',
```
Comment (bcabanes) [inline nx-dev/nx-dev/tailwind.config.js:14]: Can we use `hsla` values for consistency?

File: nx-dev/nx-dev/pages/index.tsx [hunk 2]
```diff
@@ -139,8 +139,22 @@ export function Index() {
                 </div>
                 <div className="w-full sm:w-auto pt-4 sm:pt-0">
                   <Link href="/conf">
-                    <a className="flex items-center justify-center px-4 py-2 border border-transparent rounded-md shadow-sm text-sm font-medium text-blue-nx-base bg-white">
+                    <a className="flex items-center justify-center px-4 py-2 border border-transparent group rounded-md text-md text-white font-extrabold">
                       Learn more
+                      <span className="group-hover:mr-2">
```
Comment (bcabanes) [inline nx-dev/nx-dev/pages/index.tsx:144]: This hover on the margin needs to enable a rule that we actually do not need. We can apply the same effect used in the conf page here:
```
transform-gpu transition ease-out duration-200 group-hover:translate-x-2
```
https://github.com/nrwl/nx/blob/master/nx-dev/nx-dev/pages/conf.tsx#L315

File: nx-dev/nx-dev/tailwind.config.js [hunk 3]
```diff
@@ -43,7 +41,9 @@ module.exports = {
     },
   },
   variants: {
-    extend: {},
+    extend: {
+      marginRight: ['hover'],
```
Comment (bcabanes) [inline nx-dev/nx-dev/tailwind.config.js:45]: With the updated rule above, we can remove this custom extension.

**Review by bcabanes — CHANGES_REQUESTED** [review]

File: nx-dev/nx-dev/tailwind.config.js (hunk 1 above)
Comment (meeroslav) [reply nx-dev/nx-dev/tailwind.config.js:14]: Sure. What was the reason for using HSL (and additionally ALPHA, since we don't seem to use it)?

**Pull request merged**